filter_vecs.cc

```diff
@@ -193,63 +199,67 @@ FilterVecs& FilterVecs::Instance()
   return instance;
 }
 
-Filter* FilterVecs::find_filter_vec(const QString& vecname)
+void FilterVecs::prepare_filter(const fltinfo_t& fltdata)
 {
-  QStringList options = vecname.split(',');
-  if (options.isEmpty()) {
-    fatal("A filter name is required.\n");
-  }
-  const QString svecname = options.takeFirst();
-
-  for (const auto& vec : d_ptr_->filter_vec_list) {
-    if (svecname.compare(vec.name, Qt::CaseInsensitive) != 0) {
-      continue;
-    }
+  QVector<arglist_t>* args = fltdata->get_args();
 
-    QVector<arglist_t>* args = vec.vec->get_args();
+  Vecs::validate_options(fltdata.options, args, fltdata.fltname);
 
-    Vecs::validate_options(options, args, vec.name);
+  /* step 1: initialize by inifile or default values */
```

Comment:
I really wonder if anyone uses the inifile stuff... (This isn't the place for tackling that. Just musing.)